### PR TITLE
tts: 0.0.12 -> 0.0.13

### PR DESCRIPTION
**Pull request by mweinelt**
###### Motivation for this change

https://github.com/coqui-ai/TTS/releases/tag/v0.0.13

```
❯ ./result/bin/tts --text "To help with the large amounts of pull requests, we would appreciate your reviews of other pull requests, especially simple package updates. Just leave a comment describing what you have tested in the relevant package/service. Reviewing helps to reduce the average time-to-merge for everyone. Thanks a lot if you do\!"
 > Downloading model to /home/hexa/.local/share/tts/tts_models--en--ljspeech--tacotron2-DDC
 > Downloading model to /home/hexa/.local/share/tts/vocoder_models--en--ljspeech--hifigan_v2
 > Using model: Tacotron2
 > Generator Model: hifigan_generator
Removing weight norm...
 > Text: To help with the large amounts of pull requests, we would appreciate your reviews of other pull requests, especially simple package updates. Just leave a comment describing what you have tested in the relevant package/service. Reviewing helps to reduce the average time-to-merge for everyone. Thanks a lot if you do!
 > Text splitted to sentences.
['To help with the large amounts of pull requests, we would appreciate your reviews of other pull requests, especially simple package updates.', 'Just leave a comment describing what you have tested in the relevant package/service.', 'Reviewing helps to reduce the average time-to-merge for everyone.', 'Thanks a lot if you do!']
 > Processing time: 9.314172506332397
 > Real-time factor: 0.4426849905259117
 > Saving output to tts_output.wav
```

https://shells.darmstadt.ccc.de/~hexa/tts-0.0.13.wav

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

**Comment by davidak** [comment]
Result of `nixpkgs-review pr 123016` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package built:</summary>
  <ul>
    <li>tts</li>
  </ul>
</details>

File: pkgs/tools/audio/tts/default.nix [hunk 1]
```diff
@@ -12,61 +12,41 @@
...
-    sed -i -e 's!bokeh==[^"]*!bokeh!' requirements.txt setup.py
-    sed -i -e 's!numba==[^"]*!numba!' requirements.txt setup.py
-    sed -i -e 's!numpy==[^"]*!numpy!' requirements.txt setup.py
-    sed -i -e 's!umap-learn==[^"]*!umap-learn!' requirements.txt setup.py
-    # Not required for building/installation but for their development/ci workflow
-    sed -i -e '/black/d' requirements.txt
-    sed -i -e '/isor/d' requirements.txt
-    sed -i -e '/pylint/d' requirements.txt
-    sed -i -e '/cardboardlint/d' requirements.txt setup.py
+    sed -i -e 's!librosa==[^"]*!librosa!' requirements.txt
```
Comment (davidak) [inline pkgs/tools/audio/tts/default.nix:32]: could this get upstreamed?

e.g. they say `librosa==0.7.2` is needed but we have `0.8.0` and it still works

**Comment by davidak** [comment]
I tested also the 2 new models. Here are examples: [tts_output.zip](https://github.com/NixOS/nixpkgs/files/6488498/tts_output.zip)

**Review by davidak — APPROVED** [review]
Changes look good

Builds, works

File: pkgs/tools/audio/tts/default.nix (hunk 1 above)
Comment (Mic92) [reply pkgs/tools/audio/tts/default.nix:32]: The thing is we don't really want this to be pinned in any way as it will just cause problems in future.

Comment (mweinelt) [reply pkgs/tools/audio/tts/default.nix:32]: They could set lower bounds, but I don't think they will be receptive to our needs tbh.

**Pull request merged**